app/src/main/res/values/string-not_translatable.xml

```diff
@@ -54,6 +54,9 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="editor_basic_color_scheme_gruvbox" translatable="false">Gruvbox</string>
     <string name="editor_basic_color_scheme_greenscale" translatable="false">Greenscale</string>
     <string name="editor_basic_color_scheme_sepia" translatable="false">Sepia</string>
+    <string name="unordered_list_char_dash">Dash (-)</string>
+    <string name="unordered_list_char_asterisk">Asterisk (*)</string>
+    <string name="unordered_list_char_plus">Plus (+)</string>
```

Comment:
These 3 are in not_translatable, but they are translatable right? -> move it to normal xml

Reply:
Thanks - I've move them.